### PR TITLE
Add balamod docs

### PR DESCRIPTION
**Pull request by magicwenli**
Add balamod docs based on Github Page and mdBook.

The Github Action have been test. [Actions](https://github.com/magicwenli/balamod/actions)

Pages branch needs to be set to gh-pages.

[Preview rendered page](https://magicwenli.github.io/balamod/)

### CODE REVIEW

File: docs/src/installation.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+```admonish warning
+
+Balamod currently doesn't work on macOS i386/amd64, but it will work on Apple Silicon(ARM64) aka M1/M2/M3.
+
+```
+
+| Platform | Download link |
+|----------|---------------|
+| Windows  | [{{ balamod.release_name_windows }}]({{ balamod.release_url_windows }}) |
```
Comment (magicwenli) [inline docs/src/installation.md:25]: Now we can keep the download link updated by github Actions.

Daily scheduled, pushs to `/docs` folder and `Release Workflow` will trigger doc building.

**Comment by sbordeyne** [comment]
This PR should belong to the balamod/balamod.github.io repo

**Comment by magicwenli** [comment]
> This PR should belong to the balamod/balamod.github.io repo

I find it's impossible to merge my fork to balamod.github.io

A blank branch is required.

**Comment by sbordeyne** [comment]
This PR can be closed I think, given that this has been moved to balamod.github.io

**Comment by sbordeyne** [comment]
I'm closing this since it's not relevant anymore (it has been put on the balamod webiste repo)

**Pull request closed without merging**